.deployignore

```diff
@@ -0,0 +1,7 @@
+# these apps are ignored for deployment
```

Comment:
Why are these ignored?

Reply:
- The dashr app is ignored as a test - actually atm all dashr apps are broken, so I've completely disabled them.
- The others appear to have some database in place - at least, according to [this](https://plotly.slack.com/archives/CN8DUM6JF/p1576252986006400). They are actually probably fine, since they apparently work on playground, and I'm assuming they are on gallery as well.
- The salesforce apps require an env var in playground that I don't know. Does this app work on gallery?

Reply:
>The others appear to have some database in place - at least, according to this. They are actually probably fine, since they apparently work on playground, and I'm assuming they are on gallery as well.

This is correct -- they should work. 

>The salesforce apps require an env var in playground that I don't know. Does this app work on gallery?
 
Yes, the env var should be defined on the gallery version of the app in the app settings!

Reply:
Re-enabled and deploying :)

Reply:
Deployed